### ISSUE TITLE
bug: cannot start on arch linux / wayland/ hyprland

### ISSUE THREAD

**Issue by JonnyLoughlin** [warpdotdev/Warp#4214]
### Describe the bug

Hi, I tried installing the pkg file from the website and when I try to run Warp, I get the following error:
```
libEGL warning: egl: failed to create dri2 screen
ERROR:             ICD associated with VkPhysicalDevice does not support GetPhysicalDeviceCalibrateableTimeDomainsKHR
zsh: IOT instruction (core dumped)  warp-terminal
```

I am running Arch/Hyprland on an nvida gpu and intel cpu.

### To reproduce

1. Download from https://app.warp.dev/get_warp

2. run `sudo pacman -U warp-terminal-v0.2024.02.20.08.01.stable_01-1-x86_64.pkg.tar.zst`

3. run `warp-terminal`

4. program crashes with error above

### Operating system

Linux

### Operating system and version

Arch kernal `linux 6.7.5.arch1-1`

### Regression

No, this bug or issue has existed throughout my experience using Warp

### Does this block you from using Warp daily?

Yes, this issue prevents me from using Warp daily.

### Is this a Warp specific issue? (i.e. does it happen in Terminal, iTerm, Kitty, etc.)

Yes, this I confirmed this only happens in Warp, not other terminals.

### Warp Internal (ignore): linear-label:b9d78064-c89e-4973-b153-5178a31ee54e

None

**Comment by TeddyHuang-00** [comment]
Exactly same issue here on my setup with Arch + Hyprland. Attach my Nvidia information here in case related:

Driver Version: 545.29.06    CUDA Version: 12.3  Nvidia RTX A5000

**Comment by JonnyLoughlin** [comment]
Just checked the discord, downgrading mesa fixed my issue

`sudo pacman -U /var/cache/pacman/pkg/mesa-1:23.3.3-1-x86_64.pkg.tar.zst`

**Comment by TeddyHuang-00** [comment]
This is most likely due to upstream issues:
https://gitlab.freedesktop.org/mesa/mesa/-/issues/10624
https://github.com/KhronosGroup/Vulkan-Loader/issues/1443

Guess downgrading mesa before they get fixed would be the only solution for now.

**Comment by regul4rj0hn** [comment]
It worked out-of-the-box for me on Arch+Hyprland.
If you are on Arch Linux why not use use the AUR e.g. `yay -S warp-terminal`?

**Comment by JonnyLoughlin** [comment]
The AUR had the same issue.  All that AUR package does is install the release package from warp so it shouldn't have made any difference.

Are you on nvidia or amd/integrated graphics?

**Comment by regul4rj0hn** [comment]
Yep, I thought as much. Just figured it was a bit weird to not use AUR (on top that it worked for me). 
GeForce RTX 4070 Ti. Good that the workaround fixed it for you anyway.

**Comment by vorporeal** [comment]
FYI @regul4rj0hn  - we provide official packages for Arch, which we would recommend using over a package from the AUR.  Installing the official package ensures you get the most timely updates, and that there are no issues with a bad/out-of-date package definition.  (I'm chatting with people in the comments on the AUR package to figure out the best way to handle this.)

**Comment by regul4rj0hn** [comment]
@vorporeal Sure. Apologies for being slow here, but the AUR script does pull the official package; so what do you mean by "official" in this context exactly? Is it available on *extra* as other terminal emulators are? If so, under what name? 
As of this moment I am searching an up-to-date pacman DB and for the life of me I cannot find it.

**Comment by TeddyHuang-00** [comment]
> Is it available on _extra_ as other terminal emulators are?

I guess they refer to this section: https://docs.warp.dev/getting-started/getting-started-with-warp#arch-linux-based-distributions

**Comment by Sophed** [comment]
Also experiencing this issue
